packages/browser/__tests__/Session.spec.ts

```diff
@@ -485,6 +497,81 @@ describe("Session", () => {
         redirectUrl: "https://some.redirect/url",
       });
     });
+
+    it("does nothing if the developer has not enabled session restore", async () => {
```

Comment:
```suggestion
    it("does nothing if the developer has not explicitly enabled silent authentication", async () => {
```

Reply:
Since the API option is `restorePreviousSession`, which is better for developers, I understand why this mentions this term too. However, I find it a little clearer to call it silent authentication from an internal point of view, but that may be because I'm just more familiar with the term.

Reply:
No makes sense, we're also using it for the function name: b3518237